osu.Game/Overlays/BeatmapSet/FavouriteButton.cs

```diff
@@ -17,14 +17,14 @@ public class FavouriteButton : HeaderButton
 
         public FavouriteButton()
         {
-            RelativeSizeAxes = Axes.Y;
 
             Container pink;
             SpriteIcon icon;
-            Children = new Drawable[]
+            AddRange(new Drawable[]

```

Comment:
why is this still required?

Reply:
The background would be invisible if I didn't change this. It would overwrite all the existing children and and I'd have to re-add the background.